ports/rp2/rp2_flash.c

```diff
@@ -48,14 +48,14 @@ static_assert(MICROPY_HW_FLASH_STORAGE_BASE + MICROPY_HW_FLASH_STORAGE_BYTES <=
 
 typedef struct _rp2_flash_obj_t {
     mp_obj_base_t base;
-    uint32_t flash_base;
-    uint32_t flash_size;
+    uint32_t start;
+    uint32_t len;
```

Comment:
I don't see a good reason to change these member names, I think it's better to just keep them as they were, they are more descriptive (and the names don't need to match the kwargs in the Flash constructor).

Reply:
I was conflicted about this. But agreed. In the case where an instance doesn't describe *the whole flash*, "start" and "len" are a little less... wrong... but it's probably more sensible to keep this change concise.